op_compatibility/first_wave_models.md

```diff
@@ -27,11 +27,11 @@
 | [mul](https://webmachinelearning.github.io/webnn/#dom-neuralnetworkcontext-mul) | | | | :heavy_check_mark: | | | [Mul](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Mul) | [Mul](https://www.tensorflow.org/xla/operation_semantics#element-wise_binary_arithmetic_operations) |
 | [relu](https://webmachinelearning.github.io/webnn/#api-neuralnetworkcontext-relu) |:heavy_check_mark: |:heavy_check_mark: |:heavy_check_mark: | | :heavy_check_mark: | | [Relu](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Relu) | Lowering to [Max](https://www.tensorflow.org/xla/operation_semantics#element-wise_binary_arithmetic_operations) |
 | [reshape](https://webmachinelearning.github.io/webnn/#api-neuralnetworkcontext-reshape) |:heavy_check_mark: |:heavy_check_mark: | :heavy_check_mark: | | | | [Reshape](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Reshape) | [Reshape](https://www.tensorflow.org/xla/operation_semantics#reshape) |
-| [sigmoid]() | | | | | :heavy_check_mark: | :heavy_check_mark: | [Sigmoid](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Sigmoid) | Lowering to [Tanh](https://www.tensorflow.org/xla/operation_semantics#element-wise_unary_functions) |
+| [sigmoid](https://webmachinelearning.github.io/webnn/#api-neuralnetworkcontext-unary) | | | | | :heavy_check_mark: | :heavy_check_mark: | [Sigmoid](https://github.com/onnx/onnx/blob/master/docs/Operators.md#Sigmoid) | Lowering to [Tanh](https://www.tensorflow.org/xla/operation_semantics#element-wise_unary_functions) |
```

Comment:
Should the link to `#api-modelbuilder-unary` after this PR landed?

Reply:
Good catch! I updated all occurrences of #api-neuralnetworkcontext to reflect the new links.

Reply:
Fixed.